packages/terra-content-container/tests/wdio/content-container-spec.js

```diff
@@ -0,0 +1,18 @@
+/* global browser, Terra */
+const viewports = Terra.viewports('medium');
+
+describe('Content Container', () => {
+  describe('Default', () => {
+    beforeEach(() => browser.url('/#/raw/tests/content-container/default-content-container'));
```

Comment:
nit: each of these beforeEach can be changed to before since we do not need to refresh the page between test steps

Reply:
Updated: https://github.com/cerner/terra-core/pull/1468/commits/9790d7179668344164cbceea5bcf83e822169d6a